collector/mocks/mock_collector.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.46.1. DO NOT EDIT.
+// Code generated by mockery v2.40.3. DO NOT EDIT.
 
```

Comment:
I think you may have ran `make generate` with the wrong version of mockery. Probably just using the wrong mockery? Mine is in my `go/bin`